lib/template/spec/spec_helper.rb

```diff
@@ -5,7 +5,7 @@
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
 #
-ENV["RACK_ENV"] = "test"
+ENV["PLINY_ENV"] = "test"
```

Comment:
Damn, I think we might need to leave this one as is, or at least set both. The reason this is here is that Sinatra actually checks it as it's being required and tweaks its error behavior based off of it. I can't recall the exact semantics, but I believe setting test allows stack traces to be properly dumped to console while running specs.